client/my-sites/site-settings/custom-content-types/portfolios.jsx

```diff
@@ -0,0 +1,82 @@
+import { ToggleControl } from '@wordpress/components';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import SupportInfo from 'calypso/components/support-info';
+
+function Portfolios( {
+	translate,
+	handleAutosavingToggle,
+	fields,
+	onChangeField,
+	isDisabled,
+	isAtomic,
+} ) {
+	const name = 'jetpack_portfolio';
+	const numberFieldIdentifier = name + '_posts_per_page';
+	return (
+		<FormFieldset>
+			<SupportInfo
+				text={ translate(
+					'Adds the Portfolio custom post type, allowing you to ' +
+						'manage and showcase projects on your site.'
+				) }
+				link={
+					isAtomic
+						? 'https://wordpress.com/support/portfolios/'
+						: 'https://jetpack.com/support/custom-content-types/#portfolios'
```

Comment:
This link seems incorrect, because `#privacy` will be added to it it will look like this: `#portfolios#privacy`.